test/unit/bson_import.test.js

```diff
@@ -87,3 +87,13 @@ describe('When importing BSON', function () {
     testTypes();
   });
 });
+
+describe('MongoDB export', () => {
+  const mongodb = require('../../src');
+  it('should include ObjectId', () =>
+    expect(mongodb).to.have.property('ObjectId').that.is.a('function'));
+  it('should include ObjectID', () =>
+    expect(mongodb).to.have.property('ObjectID').that.is.a('function'));
+  it('should have ObjectID and ObjectId equal each other', () =>
+    expect(mongodb.ObjectId).to.deep.equal(mongodb.ObjectID));
```

Comment:
Should this be just regular equal since we're actually expecting reference equality?
```suggestion
    expect(mongodb.ObjectId).to.equal(mongodb.ObjectID));
```

Reply:
Also, I don't think there should be any package lock changes for this PR

Reply:
Lock file changes fixed, and no longer checking deep equality 🚀